oidc-controller/api/routers/presentation_request.py

```diff
@@ -27,9 +28,24 @@
 async def send_connectionless_proof_req(
     pres_exch_id: str, req: Request, db: Database = Depends(get_db)
 ):
-    """QR code that is generated should a url to this endpoint, which responds with the
-    specific payload for that given agent/wallet"""
-    logger.info("Scanning Application headers:: " + str(req.headers))
+    """
+      If the user scanes the QR code with a mobile camera,
+      they will be redirected to a help page.
+    """
+    # First prepare the response depending on the redirect url
+    if settings.CONTROLLER_CAMERA_REDIRECT_URL is None:
+        response = HTMLResponse("Please scan with a digital wallet")
```

Comment:
Could we just respond with the default `wallet_howto.html` if the envvar is not set to a custom value? This would simplify a bit the logic, since we ship the default file no matter what.

Reply:
Yeah sure... That makes more sense.